### DISCUSSION TITLE
Consider alternative means of donations

### DISCUSSION THREAD

**Discussion by primeroz** [Blackymas/NSPanel_HA_Blueprint#884] [General]
I really want to support this project , but i so don't want to get involved with paypal 

have you considered alternative way to donate ? 

a few that comes to my mind:

* bitcoin
* kofi
* buymeacoffe

:pray: thanks again for this amazing project

**Comment by primeroz** [comment]
@edwardtfn given the amount of work you put in this ... i am going to renew my request ?

**Reply by edwardtfn** [reply]
I won't complain if you wanna [buy me an ice cream](https://www.buymeacoffee.com/edwardfirmo), but I'm having fun with this project, so there's no need for that. 😉🍦
About the alternative method for donations to the project, I will leave to @Blackymas to answer. 🙂

**Comment by Blackymas** [comment]
@primeroz 
Thank you for wanting to support us, unfortunately this is rare :)

At the moment we are not planning to introduce another method.

However, you can also make a donation without Paypal via credit card using the Paypal link

![Bildschirmfoto 2024-01-25 um 13 51 03](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/41958506/c17d4d4f-ea26-4f6d-86ac-416c824a7ee9)

**Comment by primeroz** [comment]
done , will close the discussion ... but i still think you should add something other than paypal , it would make you feel better trust me !
